torch/_inductor/codegen/triton.py

```diff
@@ -2928,7 +2928,7 @@ def define_kernel(self, src_code, node_schedule):
             compile_wrapper = IndentedBuffer()
             compile_wrapper.writeline(f"async_compile.triton({subs_name!r}, '''")
             compile_wrapper.splice(src_code, strip=True)
-            compile_wrapper.writeline("''')")
+            compile_wrapper.writeline(f"''', \"{V.graph.scheduler.current_device.type}\")")
```

Comment:
The Triton community is refactoring the API. Please double-check if the change is still workable. By the way, it is better to add some comments here.